### PR TITLE
Add support for National Delivery Zuora credits

### PR DESCRIPTION
**Pull request by shtukas**
Here we update DeliveryCreditProcessor and HolidayCreditProduct to add support for Newspaper - National Delivery.

Work done together with @johnduffell

### CODE REVIEW

File: handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/HolidayCreditProduct.scala [hunk 1]
```diff
@@ -65,6 +71,13 @@ object HolidayCreditProduct {
       productRatePlanChargeId = "2c92c0f8736c34cb01737160e5fa69e0",
       ProductRatePlanChargeName,
     )
+
+    // DO NOT USE MANUALLY: Delivery-problem credit - automated - National Delivery (CODE)
+    val NationalDelivery: CreditProduct = CreditProduct(
+      productRatePlanId = "8ad08ad58aff62ad018aff6fbea80016",
+      productRatePlanChargeId = "8ad08ad58aff62ad018aff6fc5fd005e",
```
Comment (johnduffell) [inline handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/HolidayCreditProduct.scala:78]: TODO change to a new holiday credit charge instead

**Comment by shtukas** [comment]
CODE: Delivery-problem

Screenshot 2023-10-05 at 13 30 53

```
    val NationalDelivery: CreditProduct = CreditProduct(
      productRatePlanId = "8ad08ad58aff62ad018aff6fbea80016",
      productRatePlanChargeId = "8ad08ad58aff62ad018aff6fc5fd005e",
      productRatePlanChargeName = ratePlanChargeName,
    )
```

**Comment by shtukas** [comment]
CODE: Holiday credit

Screenshot 2023-10-05 at 13 37 06

```
    val NationalDelivery: CreditProduct = CreditProduct(
      productRatePlanId = "8ad08f068aff62a4018affd6efc7743c",
      productRatePlanChargeId = "8ad08f068aff62a4018affd6eff97440",
      ProductRatePlanChargeName,
    )
```

**Comment by shtukas** [comment]
PROD: Delivery-problem

Screenshot 2023-10-05 at 15 37 58

```
    val NationalDelivery: CreditProduct = CreditProduct(
      productRatePlanId = "8a1299c28aff1e73018b004582a22581",
      productRatePlanChargeId = "8a1299c28aff1e73018b004583002583",
      productRatePlanChargeName = ratePlanChargeName,
    )
```

**Comment by shtukas** [comment]
PROD: Holiday credit 

Screenshot 2023-10-05 at 15 33 01

```
    val NationalDelivery: CreditProduct = CreditProduct(
      productRatePlanId = "8a128e208aff0721018b003d0dfe59d9",
      productRatePlanChargeId = "8a128e208aff0721018b003d0e5959e0",
      ProductRatePlanChargeName,
    )
```

**Review by johnduffell — APPROVED** [review]
nice work, have you managed to test against a subscription, either manual or automated?

**Comment by shtukas** [comment]
Screenshot 2023-10-10 at 11 19 33

**Comment by shtukas** [comment]
Screenshot 2023-10-10 at 11 20 48

**Comment by shtukas** [comment]
Screenshot 2023-10-10 at 11 21 21

**Comment by shtukas** [comment]
Screenshot 2023-10-10 at 11 21 38

**Pull request merged**